src/main/scala/io/iohk/ethereum/db/dataSource/RocksDbDataSource.scala

```diff
@@ -193,10 +193,10 @@ class RocksDbDataSource(
 
       val tableCfg = new BlockBasedTableConfig()
         .setBlockSize(blockSize)
-        .setBlockCacheSize(blockCacheSize)
+        .setBlockCache(new ClockCache(blockCacheSize))
```

Comment:
Just to understand the decision, why ClockCache and not LRUCache? From what I read LRUCache has more overhead

Reply:
Exactly this - documentation says they're very similar but `ClockCache` can perform better in certain scenarios so it seem to be better option for start.